.github/workflows/cwf-integ-test.yml

```diff
@@ -13,16 +13,12 @@ name: CWF integ test
 
 on:
   workflow_dispatch: null
-  workflow_run:
-    workflows:
-      - build-all
+  push:
     branches:
       - master
       - 'v1.*'
-    types:
-      - completed
 env:
-  SHA: ${{ github.event.workflow_run.head_commit.id || github.sha }}
+  SHA: ${{ github.sha }}
```

Comment:
This now became dead code. No need to have a duplicated variable. (This would be equivalent to `A=1; B=A` and using B everywhere after.)
And in the consequence this makes something like specifying the SHA on checkout dead code. I.e. https://github.com/magma/magma/pull/14227/files#diff-6cea9de55c2b40b7e046ce2f3b096f80f3f7018ac52fa6e55ba11a6ffce4c6f7R29-R30.

Reply:
Pushed